lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1218,6 +1218,26 @@ class DecomposeAten_UnsafeViewOp : public OpRewritePattern<Aten_UnsafeViewOp> {
 };
 } // namespace
 
+namespace {
+// Decompose constant tensor like ops.
+template <typename OpTy>
```

Comment:
this can be replaced with `template <typename OpTy, typename NewOpTy>`. The if branches below can be removed and become
```
rewriter.replaceOpWithNewOp<NewOpTy>(op, op.getType(), op.size(),
                                               op.dtype(), op.layout(),
                                               op.device(), op.pin_memory());
```